README.md

```diff
@@ -1,3 +1,56 @@
-# Northcoders News API
+# Gary-Huynh's nc-news-api
+
+This project is used to simulate a news website with functionality such as bringing up articles, browsing comments, looking for users, adding/changing/deleting articles, comments, topics and includes a voting system for both articles and comments.
+
+## Installation
+First clone the project into your local machine and then install the required dependencies.
+
+### To run this project with full functionality including testing you will need to install the following:
+
+PSQL - https://www.postgresql.org/download/
+
+dotenv - https://www.npmjs.com/package/dotenv
+
+express - https://www.npmjs.com/package/express
+
+node-postgres -  https://www.npmjs.com/package/pg
+
+jest - https://www.npmjs.com/package/jest
+
+jest-sorted - https://www.npmjs.com/package/jest-sorted
+
+pg-format - https://www.npmjs.com/package/pg-format
+
+supertest - https://www.npmjs.com/package/supertest
+
+husky - https://www.npmjs.com/package/husky
+
+
+## Usage
+
+### For initial setup after installing required dependencies please do the following
+
+create the proper environment files in the root including 
+
+.env.development
+
+.env.production
+
+.env.test
+
+
+each of which should contain the appropriate paths and then run the following commands.
```

Comment:
split this up into two separate instructions. 
-each of these files should contain the appropriate paths such as 'PGDATABASE=nc_news_test' 
-Then, run the following commands: